src/models/hooks/resource.js

```diff
@@ -26,6 +27,7 @@ const afterUpdate = async (sequelize, instance, options) => {
 };
 
 const afterCreate = async (sequelize, instance, options) => {
+  addGetResourceMetadataToQueue(instance.id, instance.url);
```

Comment:
Should we run this in afterUpdate? (Maybe only if the existing title is null.) 
It may resolve the need the run a script to batch update the data if all the commonly used URLs (the ones that are displayed) are being updated over time.

Reply:
You mean in addition to afterCreate right?

Reply:
yes

Reply:
This has been changed also.